build-tools/packages/build-tools/src/fluidBuild/options.ts

```diff
@@ -27,10 +27,8 @@ interface FastBuildOptions extends IPackageMatchedOptions, ISymlinkOptions {
 	depcheck: boolean;
 	force: boolean;
 	install: boolean;
-	nohoist: boolean;
```

Comment:
Isn't there some help text somewhere that should be updated for the removed flags?

Reply:
It was a hidden switch, so it wasn't on any help.